apps/web/.env.example

```
@@ -0,0 +1,34 @@
+# Copy to .env and fill in. .env is gitignored.
+
+# Auth — min 32-char random secret
+BETTER_AUTH_SECRET=
+
+# Database (MySQL DSN)
+DATABASE_URL=mysql://user:password@localhost:3306/struxav2
+
+# JWT keypair (PEM, newlines escaped as \n)
+JWT_PRIVATE_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)